scripts/globals/effects/super_buff.lua

```diff
@@ -1,16 +1,16 @@
 -----------------------------------
 -- xi.effect.SUPER_BUFF
+-- This is only used for Nidhogg
 -----------------------------------
 require("scripts/globals/status")
 -----------------------------------
 local effectObject = {}
 
 effectObject.onEffectGain = function(target, effect)
-    local power = effect:getPower()
-    target:addMod(xi.mod.ATTP, power)
-    target:addMod(xi.mod.DEFP, power)
-    target:addMod(xi.mod.MATT, power)
-    target:addMod(xi.mod.MEVA, power)
+    target:addMod(xi.mod.DMGMAGIC, -500)
+    target:addMod(xi.mod.DMGPHYS, -5000)
+    target:addMod(xi.mod.DMGRANGE, -5000)
```

Comment:
are you trying to make mob take 50% damage? Should use UDMGMAGIC, etc. instead

Reply:
-50% pdt and -5% mdt is what I was trying to do